authorization.md

```diff
@@ -259,7 +260,54 @@ matches the value supplied at registration time for the specified `client_id`).
 </ol>
  
 
-If an error is encountered during the authorization process, servers SHALL respond with errors as defined by the [OAuth 2 specification](https://tools.ietf.org/html/rfc6749#section-5.2). Servers SHOULD also include an error_uri and error_description as defined by OAuth 2.
+If an error is encountered during the authentication process, the server SHALL 
+respond with an `invalid_client` error as defined by 
+the [OAuth 2.0 specification](https://tools.ietf.org/html/rfc6749#section-5.2). 
+
+### Issuing Access Tokens
+
+Once the backend service has been authenticated, the authorization server SHALL
+mediate the request to assure that the scope requested is within the scope pre-authorized
+to the backend service.
+
+If the access token request is valid and authorized, the authorization server
+SHALL issue an access token in response.  The access token SHALL be a JSON object with 
+the following properties: 
+
+<table class="table">
+  <thead>
+    <th colspan="3">Access token response: property names</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td><code>access_token</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>The access token issued by the authorization server.</td>
+    </tr>
+    <tr>
+      <td><code>token_type</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>Fixed value: <code>bearer</code>.</td>
+    </tr>
+    <tr>
+      <td><code>expires_in</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>The lifetime in seconds of the access token. The recommended value is <code>300</code>, for a five-minute token lifetime.</td>
+    </tr>
+    <tr>
+      <td><code>scope</code></td>
+      <td><span class="label label-success">required</span></td>
+      <td>Scope of access authorized. Note that this can be different from the scopes requested by the app.</td>
+    </tr>
+  </tbody>
+</table>
+
+Access tokens issued under this profile SHALL be short-lived; the `expires_in` 
+value SHOULD NOT exceed `300`, which represents an expiration-time of five minutes.  
+
+If an error is encountered during the authorization process, the server SHALL
+respond with the appropriate error message defined in [Section 5.2 of the OAuth 2.0 specification](https://tools.ietf.org/html/rfc6749#page-45).  The server SHOULD include an 
+`error_uri` and `error_description` as defined in OAuth 2.0.  
```

Comment:
Why both uri and description? ...and aren't they optional?

Reply:
Agreed, I'd like to modify like:

```suggestion
`error_uri` or `error_description` as defined in OAuth 2.0.  
```
@bakerdb if this make sense, I think you can apply this change by taking this suggestion inline.